tests/Get-DbaDbMirror.Tests.ps1

```diff
@@ -23,7 +23,6 @@ Describe "$commandname Integration Tests" -Tag "IntegrationTests" {
 
         Remove-DbaDbMirror -SqlInstance $script:instance2, $script:instance3 -Database $db1, $db2 -Confirm:$false
         Remove-DbaDatabase -SqlInstance $script:instance2, $script:instance3 -Database $db1, $db2 -Confirm:$false
-        $null = Get-DbaDatabase -SqlInstance $script:instance2, $script:instance3 -Database $db1, $db2 | Remove-DbaDatabase -Confirm:$false
         $null = $server.Query("CREATE DATABASE $db1")
         $null = $server.Query("CREATE DATABASE $db2")
```

Comment:
While the `$null =` is not needed with `.Query()` here, I think it should be used in the first two line to prevent the output of the commands.

Reply:
agreed! i tend to add extra $nulls around in case something changes in the future